src/fluree/http_api/handlers/ledger.clj

```diff
@@ -106,7 +106,7 @@
                    did (assoc :did did)))
         query* (if opts (assoc query :opts opts) query)]
     {:status 200
-     :body   (deref! (fluree/from-query conn query* {:format format}))}))
+     :body   (deref! (fluree/query-connection conn query* {:format format}))}))
```

Comment:
This is nitpicky but I feel like this name change makes this fn confusing. I originally chose `from-query` because that was the fundamental difference from kind of query the `query` fn took. I.e. it takes a ledger in a `from` key. And since the query itself embeds the ledger (and someday ledgers, plural) to use, you have to call it on a conn.

But is there a reason I'm missing this works better (now)?

Reply:
This wasn't my change, this is updating to work with a change from https://github.com/fluree/db/pull/606, which this branch builds off of